source/draw/RasterMonochrome.ooc

```diff
@@ -49,21 +49,62 @@ RasterMonochrome: class extends RasterPacked {
 			}
 	}
 	resizeTo: override func (size: IntSize2D) -> This {
+		this resizeTo(size, TransformMethod Smooth) as This
+	}
+	resizeTo: override func ~withMethod (size: IntSize2D, method: TransformMethod) -> This {
```

Comment:
I think I had some problems with virtual functions and default values for arguments. I will try again.